Package.swift

```diff
@@ -104,10 +110,10 @@ private extension Environment {
                 .binaryTarget(
                     name: "CoreBinary",
                     url: "https://github.com/trafi/maas-components/releases/download/0.1.0-dev03/CoreBinary.xcframework.zip",
-                    checksum: "08d2bc473fe1ea73d9d389b8b491ecd378fde82c49d24bc1579804f0e8ed978b"),
+                    checksum: "419f3c06d6dc73f042794b81eb7c92e922077f11a0253aa04817fa20938dc118"),
```

Comment:
if you have uploaded a new binary, shouldn't we have used a new tag as well? dev04?

Reply:
We're uploading xcframework in to dev-03 instead of creating new one.

Reply:
We discussed that we will reuse the same tag until someone actually uses it. But it will be necessary to move the dev03 tag to this commit.

Reply:
So did i get it right we increasing tag? 🤔

Reply:
tag stays the same, but we will need to move tag to the commit after merge